### ISSUE TITLE
`no-duplicate-attr-inheritance`: should ignore components with multiple root nodes?

### ISSUE THREAD

**Issue by andreww2012** [vuejs/eslint-plugin-vue#2596]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 9.13.0
- **eslint-plugin-vue version:** 9.30.3
- **Vue version:** 3.5.12
- **Node version:** 
- **Operating System:** 

**Please show your full configuration:**

```json5

```

**What did you do?**	

If a component has multiple root nodes, Vue [encourages the presence of `v-bind="$attrs"`](https://vuejs.org/guide/components/attrs.html#attribute-inheritance-on-multiple-root-nodes) because essentially `inheritAttrs` is implicitly set to `false`. I suggest that this rule ignores such components as setting `inheritAttrs: false` manually becomes redundant. Or perhaps have an option to ignore or not ignore for the sake of explicitness?

Example of SFCs that shouldn't be reported:

```vue
<template>
  <div v-bind="$attrs"></div>
  Some text
</template>

<template>
  <div v-bind="$attrs"></div>
  <div>Another div</div>
</template>
```

[Test on the rule page](https://eslint.vuejs.org/rules/no-duplicate-attr-inheritance.html)

**What did you expect to happen?**

**What actually happened?**

**Repository to reproduce this issue**
